recipes/poretools/meta.yaml

```diff
@@ -1,24 +1,25 @@
 package:
     name: poretools
-    version: "0.5.1"
+    version: "0.6.0"
 source:
-    git_url: https://github.com/arq5x/poretools
+    fn: poretools_v0.6.0.tar.gz
+    url: https://github.com/arq5x/poretools/archive/v0.6.0.tar.gz
+    md5: 7fe9dd17b6fdefb976039c00b95ac18a
+#    git_url: https://github.com/arq5x/poretools
 
 requirements:
     build:
         - python
-        - watchdog >=0.8.3
-        - rpy2 >=2.4.2
-        - h5py >=2.0
+        - h5py >=2.2.0
 
     run:
         - python
         - hdf5 >=1.8.7
-        - r >=3.0.0
-        - rpy2 >=2.4.2
-        - h5py >=2.0
-        - watchdog >=0.8.3
+        - h5py >=2.2.0
         - qcli >=0.1.1
+        - matplotlib
+        - seaborn
+        - pandas
 
 build:
     skip: True # [not py27 or osx]
```

Comment:
It is building for me on osx with the following sections: 
<pre>
requirements:
    build:
        - python
        - setuptools
        - hdf5 >=1.8.7
        - h5py >=2.2.0
        - matplotlib
        - seaborn
        - pandas

    run:
        - python
        - hdf5 >=1.8.7
        - h5py >=2.2.0
        - matplotlib
        - seaborn
        - pandas

build:
    skip: True # [not py27]
</pre>